tests/sqllogictests/suites/udf_native/01_0001_udf_wasm.test

```diff
@@ -0,0 +1,63 @@
+query 
+select name, size, creator from list_stage(location => '@data/udf/wasm/test10-udf-wasm-gcd/') order by name
+----
+udf/wasm/test10-udf-wasm-gcd/test10-udf-wasm-gcd.tar.gz 8932 NULL
```

Comment:
Maybe we just need this compress file in git.

Seems the wasm cost 2.8 MB spaces in git.

We can use:

```
CompressAlgorithm::from_extension() to get the `Option<CompressAlgorithm>` and decode it.

// I apologize for the previous mistake or confusion.
```

Reply:
Good one. The necessary changes to meet the requirements have been completed, and PR is now ready for review.